parcels/particlefile.py

```diff
@@ -28,7 +28,8 @@ class ParticleFile(object):
 
     :param name: Basename of the output file
     :param particleset: ParticleSet to output
-    :param user_vars: A list of additional user defined particle variables to write
+    :param user_vars: A list of additional user defined particle variables to write for all particles and all times
+    :param user_vars_once: A list of additional user defined particle variables to write for all particle only once at initial time
```

Comment:
Actually, these `users_vars` and `user_vars_once` are not parameters at all (they are not input in the `__init__`), so not sure why these lines were even here. Perhaps better to write brief comments on line 93 and 94 when `self.user_vars = []`?